karspexet/ticket/templates/seat_map.svg

```diff
@@ -0,0 +1,16 @@
+<svg viewBox="0 0 480 440" xmlns="http://www.wd.org/2000/svg">
+  <rect x="0" y="400" width="480" height="40" stroke-width="1" fill="none" stroke="black" class="stage"/>
+  <text x="220" y="420" fill="black" font-size="14" text-anchor="start">Scen</text>
```

Comment:
I tried around a bit with this, and it does not seem to work for positioning and size (so not for `x/y/height/width`, but the rest of them I managed to extract to CSS.